.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Fixes #<!-- If this addresses a specific issue, please provide the issue number here -->
+
+- [ ] CHANGELOG.md updated
+
+## Description
+<!-- Please discuss the changes you have worked on. What do the changes do; why is this PR needed? -->
+
+## Specific Changes
```

Comment:
Could be replaced with "## Changelog" and devs can just copy-and-paste their changelog here, to lower their workload.